.bash_profile

```diff
@@ -15,12 +15,13 @@ if [ -f "$(brew --prefix)/opt/bash-git-prompt/share/gitprompt.sh" ]; then
   __GIT_PROMPT_DIR=$(brew --prefix)/opt/bash-git-prompt/share
     source "$(brew --prefix)/opt/bash-git-prompt/share/gitprompt.sh"
 fi
-export PATH="/Users/spagolu/nvim/bin:$PATH"
-export EDITOR="/Users/spagolu/nvim/bin/nvim"
+export EDITOR="/usr/local/bin/nvim"
```

Comment:
Parsing the request body could be DRY'd out by extracting that into a utility method

`def request_body(request)

end`